Tasks/Common/utility-common/kubectlutility.ts

```diff
@@ -109,17 +109,23 @@ function getExecutableExtention(): string {
 
 export async function getAvailableKubectlVersions() {
     var request = new WebRequest();
-    request.uri = "https://api.github.com/repos/kubernetes/kubernetes/releases";
     request.method = "GET";
-
-    try {
-        var response = await sendRequest(request);
-        let versions = [];
-        response.body.forEach(release => {
-            versions.push(release["tag_name"])
-        });
-        return versions;
-    } catch (error) {
-        throw error;
+    let page_number = 0;
+    let versions = [];
+    while (true) {
+        try {
+            request.uri = `https://api.github.com/repos/kubernetes/kubernetes/releases?page=${page_number}&per_page=100`;
+            var response = await sendRequest(request);
+            if (response.body.length == 0) {
+                break;
+            }
+            response.body.forEach(release => {
+                versions.push(release["tag_name"])
```

Comment:
i would recommend to check this property is nonnull or empty, then add.

should we check for prerelease property also? we will show only released or should we show all?

Reply:
I suppose it should be fine to allow pre-released versions as well? The user would be consciously making that choice.